pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=41.2.0", "wheel", "setuptools_scm>3"]
+
+
+[tool.setuptools_scm]
```

Comment:
Oh cool. Wasn't this the main reason why we still have a `setup.py` in pytest?

Reply:
Yes, it was added very recently to setuptools